service.py

```diff
@@ -0,0 +1,41 @@
+class InventoryService:
+
+    inventory = {}
+
+    def __init__(self):
+        self.inventory = InventoryService.inventory

```

Comment:
Just self.inventory works … no need to cal a class by its name inside the seamen class